.circleci/config.yml.in

```diff
@@ -263,14 +263,24 @@ jobs:
   prototype_test:
     docker:
       - image: circleci/python:3.7
+    resource_class: xlarge
     steps:
       - run:
           name: Install torch
-          command: pip install --user --progress-bar=off --pre torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
+          command: |
+            pip install --user --progress-bar=off numpy
+            pip install --user --progress-bar=off --pre torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
       - run:
           name: Install prototype dependencies
           command: pip install --user --progress-bar=off git+https://github.com/pytorch/data.git
       - checkout
+      - run:
+          name: Download model weights
+          background: true
+          command: |
+            sudo apt update -qy && sudo apt install -qy parallel wget
+            python scripts/collect_model_urls.py torchvision/prototype/models \
+                | parallel -j0 wget --no-verbose -P ~/.cache/torch/hub/checkpoints {}
```

Comment:
This will collect all model URLs and download them in parallel. The system utilities `parallel` and `wget` are available through `conda` so we can simply install them there instead of using `apt` which only works on Ubuntu.

Reply:
Yes we should make this cross-platform. If we adopt this solution for running the tests of models with proper weights, we will need to do this across platforms and cpu/gpu.